app/views/notifications/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= simple_form_for(@notification) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.association :event %>
+    <%= f.input :status, as: :select, collection: Notification::VALID_STATUSES %>
```

Comment:
This is technically correct and I like how you accomplished it, but status should probably be managed by the controller. People can probably only choose **Pending** **Active** and **Canceled**. Then there is a bit of trickiness around changing status. A new Notification can only be changed to pending or active. A pending one can only move to active or canceled. You could capture the state machine in a hash, using the current state as the key, and the possible transitions as the values ?